metomi/rosie/suite_id.py

```diff
@@ -456,6 +404,77 @@ def _from_location(self, location):
         if "commit:revision" in info_entry:
             self.revision = info_entry["commit:revision"]
 
+    @staticmethod
+    def _find_vc_file_from_location(location):
+        """Search a location and parents for a version control log file.
+
+        Args:
+            location: Path to search.
+
+        Returns:
+            If a file is found, a path, else None.
+        """
+        suite_engine_proc = SuiteEngineProcessor.get_processor()
+        suite_dir_rel_root = getattr(
+            suite_engine_proc, "SUITE_DIR_REL_ROOT", None
+        )
+        loc = Path(location).expanduser().resolve()
+        sdrr = Path('~', suite_dir_rel_root).expanduser().resolve()
```

Comment:
I think this line might break with traceback if you get the default None back from `suite_dir_rel_root`. Maybe put in try/except?

Reply:
```suggestion
        if suite_dir_rel_root:
            sdrr = Path('~', suite_dir_rel_root).expanduser().resolve()
        else:
            return None
```